debian/dell-recovery.templates

```diff
@@ -64,6 +64,30 @@ Template: ubiquity/text/hdd_warning_label
 Type: text
 _Description: WARNING: All personal files and changes will be lost.
 
+Template: ubiquity/text/dhc_hdd_recovery
+Type: text
+_Description: Restore Linux OS partitions with DHC
```

Comment:
This is a customer facing string, right?  No one will know what DHC means.  I think you need to expand the acronym.